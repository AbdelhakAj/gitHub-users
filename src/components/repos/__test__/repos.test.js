import React from "react";
import renderer from "react-test-renderer";
import Repos from "../repos";

describe("<Repos />", () => {
  it("renders repos", () => {
    const component = renderer.create(
      <Repos data={{ id: "", name: "", fullName: "" }} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
