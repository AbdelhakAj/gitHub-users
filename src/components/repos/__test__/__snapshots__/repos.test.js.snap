// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Repos /> renders repos 1`] = `
<div
  className="repos"
>
  <h2
    className="repos-name"
  >
    
  </h2>
  <span
    className="repos-fullname"
  >
    Full Name: 
    
  </span>
  <span
    className="repos-id"
  >
    id :
    
  </span>
</div>
`;
